<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1100</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>19</width>
      <height>16</height>
     </size>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1195</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_2"/>
    <addaction name="actionSave"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
    <addaction name="actionSave_all"/>
    <addaction name="actionClose_all"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionSelect_all"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_folder"/>
    <addaction name="actionShow_second_thing"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave_2"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_all"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionCut"/>
   <addaction name="actionSelect_all"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionPrint"/>
   <addaction name="actionClose_all"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>512</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1191</width>
       <height>111</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-add-file-32.png</normaloff>:/new/imgs/Res/icons8-add-file-32.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-open-view-32.png</normaloff>:/new/imgs/Res/icons8-open-view-32.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-save-as-32.png</normaloff>:/new/imgs/Res/icons8-save-as-32.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-exit-32.png</normaloff>:/new/imgs/Res/icons8-exit-32.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-copy-32.png</normaloff>:/new/imgs/Res/icons8-copy-32.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-paste-32.png</normaloff>:/new/imgs/Res/icons8-paste-32.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-cut-32.png</normaloff>:/new/imgs/Res/icons8-cut-32.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-undo-32.png</normaloff>:/new/imgs/Res/icons8-undo-32.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-redo-32.png</normaloff>:/new/imgs/Res/icons8-redo-32.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-print-32.png</normaloff>:/new/imgs/Res/icons8-print-32.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-save-32.png</normaloff>:/new/imgs/Res/icons8-save-32.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-save-all-32.png</normaloff>:/new/imgs/Res/icons8-save-all-32.png</iconset>
   </property>
   <property name="text">
    <string>Save all</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-close-all-tabs-32.png</normaloff>:/new/imgs/Res/icons8-close-all-tabs-32.png</iconset>
   </property>
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-select-all-32.png</normaloff>:/new/imgs/Res/icons8-select-all-32.png</iconset>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
  <action name="actionShow_folder">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-view-details-32.png</normaloff>:/new/imgs/Res/icons8-view-details-32.png</iconset>
   </property>
   <property name="text">
    <string>Show catalog</string>
   </property>
  </action>
  <action name="actionShow_second_thing">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/imgs/Res/icons8-moleskine-32.png</normaloff>:/new/imgs/Res/icons8-moleskine-32.png</iconset>
   </property>
   <property name="text">
    <string>Show reviewer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
